Source/SwiftLintFramework/Rules/RuleConfigurations/FileHeaderConfiguration.swift

```diff
@@ -38,9 +38,9 @@ public struct FileHeaderConfiguration: RuleConfiguration, Equatable {
     private static let defaultRegex = regex("\\bCopyright\\b", options: [.caseInsensitive])
 
     public var consoleDescription: String {
-        return severityConfiguration.consoleDescription + ", required_string: \(requiredString)" +
-            ", required_pattern: \(requiredPattern), forbidden_string: \(forbiddenString)" +
-            ", forbidden_pattern: \(forbiddenPattern)"
+        return severityConfiguration.consoleDescription + ", required_string: \(String(describing: requiredString))" +
```

Comment:
We actually already have a bug here as it prints `Optional(actual value)`. Can you please change these to have default values (and unwrap the optional if possible)?